docs/README.md

```diff
@@ -1,41 +1,42 @@
+[![GitHub License](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](http://www.apache.org/licenses/LICENSE-2.0)
 
-This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
+# ADIDAS MVI LIBRARY
 
-### Installation
+## Introduction
+ADIDAS MVI is an efficient library that helps you implement the MVI architectural pattern in your Android or JVM applications. MVI is a pattern that promotes a unidirectional flow of data and provides a clear separation between the different components of your application. This library aims to simplify the implementation of MVI by providing a set of tools and utilities.
 
-```
-$ yarn
-```
-
-### Local Development
-
-```
-$ yarn start
-```
+## Features
+Simple and lightweight: The library is designed to be easy to understand and use, without unnecessary complexity or overhead.
+Unidirectional data flow: The MVI pattern ensures that data flows in a single direction, making it easier to reason about and test your application.
+Separation of concerns: The pattern encourages a clear separation between the model, view, and intent components of your application.
+Immutable data: The library promotes the use of immutable data structures, which helps reduce bugs and makes your code more reliable.
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+## Usage in adidas
+MVI library is now being used in adidas CONFIRMED app. In fact, this library was developed initially as core architecture of CONFIRMED app and later it was decided to open source it.
 
-### Build
+## Installation
+To use the ADIDAS MVI Library in your Android project, follow these steps:
 
+Open the build.gradle file of your project.
+Add the following dependency to the dependencies block:
 ```
-$ yarn build
+implementation("com.adidas.mvi:mvi:{LATEST_VERSION}")
 ```
+Sync your project with the Gradle files.
+## Usage
+To use the ADIDAS MVI Library in your application, you need to understand the basic concepts and components of the MVI pattern.
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
-
-### Deployment
-
-Using SSH:
-
-```
-$ USE_SSH=true yarn deploy
-```
-
-Not using SSH:
-
-```
-$ GIT_USER=<Your GitHub username> yarn deploy
-```
+## Concepts
+Model: The model represents the state of your application. It's an immutable data structure that holds all the information needed to render the UI.
+View: The view is responsible for rendering the UI and reacting to user interactions. It observes the model and updates the UI accordingly.
+Intent: An intent represents a user action or an event that triggers a state change in the model. Intents are dispatched to the model, which updates its state based on the intent received.
+## Components
```

Comment:
Side Effect is not a 'concept' in MVI, it's an implementation detail, that's why I added it to Components.

Reply:
yes, its not a concept in the by the book MVI but we could be more concrete since the library is an implementation of the MVI.

i'm fine with only having it on components